### PR TITLE
[DTRA] Ahmad /FEQ-1739/ Removing unused js

### PR DESCRIPTION
**Pull request by ahmadtaimoor-deriv**
# Description
The card removes all the unused js from the smartcharts package

### CODE REVIEW

**Review by review-deriv — COMMENTED** [review]
PR description is too short. You can use this format:

```
# Description
This is a summary of the changes. This should describe why do we need this change.
Whys and Hows come here.

# Cards
* https://clickup.com/12345
* https://clickup.com/12346

```

**Review by farhan-nurzi-deriv — APPROVED** [review]

File: app/connection/NetworkMonitor.ts [hunk 1]
```diff
@@ -40,9 +40,6 @@ class NetworkMonitor {
                 this.establishConnection();
             });
             window.addEventListener('offline', () => this.setNetworkStatus('offline'));
-        } else {
-            // default to always online and fallback to WS checks
-            navigator.onLine = true;
```
Comment (maryia-deriv) [inline app/connection/NetworkMonitor.ts:45]: @ahmadtaimoor-deriv please elaborate why don't we need this anymore? :)

Comment (ahmadtaimoor-deriv) [reply app/connection/NetworkMonitor.ts:45]: `navigator.onLine = true;`

navigator.onLine is readonly and cannot be explicitly set. It is controlled by the browser. Setting onLine to true will not have any effect . 
https://developer.mozilla.org/en-US/docs/Web/API/Navigator/onLine

**Review by kate-deriv — APPROVED** [review]

File: app/connection/NetworkMonitor.ts (hunk 1 above)
Comment (ahmadtaimoor-deriv) [reply app/connection/NetworkMonitor.ts:45]: https://www.w3schools.com/jsref/prop_nav_online.asp (it says it is readonly)
@maryia-deriv , I think it needs to be removed from the deriv-app as well. Setting a readonly property to anything will not have any affect . 
Can also try changing it's value in a code snippet
https://www.w3schools.com/jsref/tryit.asp?filename=tryjsref_nav_all
I confirmed it with quality team as well. I'll create a PR to remove it from deriv-app as well.

**Review by maryia-deriv — APPROVED** [review]

**Review by kate-deriv — APPROVED** [review]

**Review by henry-deriv — APPROVED** [review]

**Pull request closed without merging**